packages/deck/frontend/sirius-components-deck/src/representation/useDeckMutations.ts

```diff
@@ -259,7 +237,7 @@ export const useDeckMutations = (editingContextId: string, representationId: str
   const [rawcreateCard, rawCreateCardResult] = useMutation<GQLCreateCardData, GQLCreateCardVariables>(
     createCardMutation
   );
-  useErrorReporting(rawCreateCardResult, (data) => data?.createCard);
+  useReporting(rawCreateCardResult, (data: GQLCreateCardData) => data.createCard);
```

Comment:
This is wrong, it's not your fault, the error has been on master for quite some time. We have a simple pattern where everything follows a logic yet people want to change things and they make mistakes. The name of the mutation is not `createCard` so the name of the payload, type etc should change. Because of that, here we are extracting a payload which do not exist. The existing code either did not properly execute the code or made incorrect assumptions.